tests/bin/test_tmux_layout_integration.py

```
@@ -1,289 +1,289 @@
+        tmppath, test_session = self.setup_env()
+        
+        try:
+            # tmuxセッションを作成
+            busd.ensure_session()
+            time.sleep(0.5)
+            
+            # 作業ディレクトリを作成
+            for task in ["PMAI", "T001", "T002", "T003"]:
+                (tmppath / "work" / task).mkdir(parents=True)
+            
+            # PMAIを起動（左上ペインに配置）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)